backend/uclapi/dashboard/templates/dashboard.html

```diff
@@ -18,7 +18,7 @@
 
   <script>
     window.initialData = {{ initial_data|safe }};
-    window.staticURL = {% static "" %}
+    window.staticURL = "{% static "" %}";
```

Comment:
Why are we doing this exactly? What is the reason for quoting it I mean?

Reply:
Full URLs break the rendering of React without putting the full URL in quotes.